tasking/pulp/tasking/celery_app.py

```diff
@@ -92,6 +94,7 @@ def get_resource_manager_lock(name):
     :param name:   The hostname of the worker
     :type  name:   basestring
     """
+    from pulp.app.models.task import TaskLock, Worker
```

Comment:
Another possible option would be to shorten an import at the module level to `from pulp import app as pulp_app` and refer to the modules in this method the same way as identified ^.